vertx-mysql-client/src/main/java/io/vertx/mysqlclient/impl/protocol/ColumnDefinition.java

```diff
@@ -17,77 +17,30 @@
 import java.sql.JDBCType;
 
 public final class ColumnDefinition implements ColumnDescriptor {
+
   /*
     https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_com_query_response_text_resultset_column_definition.html
    */
-  private final String catalog;
-  private final String schema;
-  private final String table;
-  private final String orgTable;
   private final String name;
```

Comment:
add a comment saying that this is a subset of the data that we actually need in practice